code/__DEFINES/dcs/signals.dm

```diff
@@ -85,6 +85,7 @@
 #define COMSIG_ATOM_LIGHTEATER_ACT "atom_lighteater_act"		//! from base of atom/light_eater_act(): (obj/item/light_eater/light_eater)
 #define COMSIG_ATOM_EMINENCE_ACT "atom_eminence_act"				//! from base of atom/eminence_act(): ()
 #define COMSIG_ATOM_RCD_ACT "atom_rcd_act"						//! from base of atom/rcd_act(): (/mob, /obj/item/construction/rcd, passed_mode)
+#define COMSIG_ATOM_USED_IN_CRAFT "atom_used_in_craft"			///from base of atom/CheckParts(): (atom/movable/new_craft) - The atom has just been used in a crafting recipe and has been moved inside new_craft.
```

Comment:
I don't think that's a thing in our codebase? This PR seems to be registering this in multiple places, but not actually invoking.

Reply:
Updated, removed this signal.

Reply:
Now I'm angry you included an unrelated change to envirosuit helmet light in the same commit 😠

Reply:
To be honest, i'm split on that one, since it either give envirohemlet virtually speaking a flat upgrade compared to other helmets, and really, for EVA suit it's annoying, again this is why I want to TM this first to see player feedback to tweak before merge